src/duckstation-qt/inputbindingwidgets.cpp

```diff
@@ -61,19 +60,36 @@ void InputBindingWidget::initialize(SettingsInterface* sif, InputBindingInfo::Ty
   reloadBinding();
 }
 
-void InputBindingWidget::updateText()
+void InputBindingWidget::updateElidedText()
+{
+  // https://github.com/qt/qtbase/blob/5dbb3b358950726447765e4fea7feb040a303860/src/widgets/styles/qcommonstyle.cpp#L4867
+  const int button_margin = style()->pixelMetric(QStyle::PM_ButtonMargin, nullptr, this);
+  const int frame_width = style()->pixelMetric(QStyle::PM_DefaultFrameWidth, nullptr, this);
+  const int text_width = std::max(1, width() - button_margin - frame_width * 2);
+  QString elided = fontMetrics().elidedText(m_full_text, Qt::ElideMiddle, text_width);
+
+  // fix up accelerators
+  if (elided.contains('&'))
```

Comment:
was wondering... why not `+` for key combos instead of `&`? it doesn't require escaping and is probably more common?

Reply:
Was several years ago so I can't remember my reasoning exactly, but can't change it now without breaking existing configs.